### ISSUE TITLE
Reignite the spark for this project.

### ISSUE THREAD

**Issue by zlatinejc** [ayastreb/money-tracker#25]
I have to say this project is very promising (at least to me), that is why I would like to see any further thoughts on it, maybe even a roadmap of some sort. 

I am willing to provide my own input into it without any hesitation.

Please leave your comments below.

**Comment by ayastreb** [comment]
hi! thanks for your interest!
There's no real roadmap since I don't have much free time for this project at the moment.
If you have something on your mind and would want to implement it, I'd be happy to see a pull request :)

**Comment by zlatinejc** [comment]
I understand, time is something we never master ;)

Well, before diving in to the PRs I think what this project needs is a proper documentation, maybe even a wiki section (installation guide, tech. stack description, dev approaches & concepts used some kind of roadmap).

BTW, what are your thoughts on the section regarding the budget ? What do you think a budget is and what did you have in mind when you were planning / brainstorming this module ?

**Comment by zlatinejc** [comment]
I see that the entities in the accounts module are already set up as tags! Great job!

Another thought that crossed my mind is that one could apply rules from the budget module into transaction types (transfer, income, expense) so that they can serve as warning signs / notifiers.

**Comment by ayastreb** [comment]
Yes, planned or scheduled transactions were also on my mind :) someone also
requested that in another ticket.

Budget notifications would be great, e.g. when you're getting close to some
threshold it would show a notice.

**Comment by zlatinejc** [comment]
I think it would be prudent to show an ERD which would explain basic entity relation dynamic made within the app, so we could understand general flows already made within the app / and plan easier in the future.
